src/south/OPCUA_HA/OPCUA_HA.class.js

```diff
@@ -226,6 +249,49 @@ class OPCUA_HA extends ProtocolHandler {
         // eslint-disable-next-line max-len
         this.logger.silly(`Read from ${opcStartTime.getTime()} to ${intervalOpcEndTime.getTime()} (${intervalOpcEndTime - opcStartTime}ms) ${nodesToRead.length} nodes [${nodesToRead[0]}...${nodesToRead[nodesToRead.length - 1]}]`)
         const options = { timeout: this.readTimeout, numValuesPerNode: this.maxReturnValues }
+        switch (scanGroup.resampling) {
+          case 'Second':
+            options.processingInterval = 1000
+            break
+          case '10 Seconds':
+            options.processingInterval = 1000 * 10
+            break
+          case '30 Seconds':
+            options.processingInterval = 1000 * 30
+            break
+          case 'Minute':
+            options.processingInterval = 1000 * 60
+            break
+          case 'Hour':
+            options.processingInterval = 1000 * 3600
+            break
+          case 'Day':
+            options.processingInterval = 1000 * 3600 * 24
+            break
+          case 'None':
```

Comment:
Shouldn't we use `0` as default?
```Interval between returned Aggregate values. The value 0 indicates that there is no ProcessingInterval defined.```
From this link: https://reference.opcfoundation.org/v104/Core/docs/Part11/6.4.4/

Reply:
I believe we should make mandatory to add a resamplingInterval but not take 0 as a default 
0, as you point out rightly, has a specific meaning (one aggregated value from start to end date) so I would not use it as a default. 
In fact, I don't think we should use it with OIBus as it break the logic with lastCompletedAt.